cardano-node/src/Cardano/Tracing/ToObjectOrphans.hs

```diff
@@ -822,6 +837,47 @@ instance ToObject (FetchDecision [Point header]) where
     mkObject [ "kind" .= String "FetchDecision results"
              , "length" .= String (pack $ show $ length results) ]
 
+instance (Show peer, Show txid, Show tx)
+    => ToObject (TraceLabelPeer peer
+         (TraceSendRecv (TxSubmission txid tx))) where
+  toObject verb (TraceLabelPeer peerid (TraceSendMsg (AnyMessage msg))) =
+    mkObject
+        [ "kind" .= String "TraceSendMsg"
+        , "peer" .= show peerid
+        , "message" .= toObject verb msg
+        ]
+  toObject verb (TraceLabelPeer peerid (TraceRecvMsg (AnyMessage msg))) =
+    mkObject
+        [ "kind" .= String "TraceRecvMsg"
+        , "peer" .= show peerid
+        , "message" .= toObject verb msg
+        ]
+
+instance (Show txid, Show tx) => ToObject (Message
+                                   (TxSubmission txid tx) from to) where
+  toObject _verb (MsgRequestTxs txids) =
+    mkObject
+        [ "kind" .= String "MsgRequestTxs"
+        , "txIds" .= String (pack $ show $ txids)
+        ]
+  toObject _verb (MsgReplyTxs txs) =
+    mkObject
+        [ "kind" .= String "MsgReplyTxs"
+        , "txs" .= String (pack $ show $ txs)
+        ]
+  toObject _verb (MsgRequestTxIds _ _ _) =
+    mkObject
+        [ "kind" .= String "MsgRequestTxIds"
+        ]
+  toObject _verb (MsgReplyTxIds _) =
+    mkObject
+        [ "kind" .= String "MsgReplyTxIds"
+        ]
+  toObject _verb MsgDone =
+    mkObject
+        [ "kind" .= String "MsgDone"
+        ]
```

Comment:
Is this going to be too noisy being logged at info?

Reply:
if these messages are useful to see in production then "Info" is right.

Reply:
Since `MsgRequestTxs` and `MsgReplyTxs` are the only useful constructors right now, I added a transformer in `Cardano.Tracing.Tracers` (see `traceMsgReqAndReplyTxs`) to ensure that those are the only ones that are traced.